msstats.py

```diff
@@ -837,15 +867,23 @@ def main():
         if pos_json.endswith(".json")
     ]
 
+    if not service_accounts:
+        print("Error: No service account JSON files found in current directory")
+        exit(1)
+
     projects = {}
     # For each service account found try to fetch the clusters metrics using the
     # google cloud monitoring api metrics
     for service_account in service_accounts:
-        project_id, stats = project_id, stats = process_google_service_account(
```

Comment:
It looks like this duplicate got destructuring was introduced recently here and can be removed.

https://github.com/Redislabs-Solution-Architects/msstats/pull/17